src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -426,8 +426,8 @@ private void GenLogMethod(LoggerMethod lm, string nestedIndentation)
                 {nestedIndentation}{enabledCheckIndentation}{level},
                 {nestedIndentation}{enabledCheckIndentation}new global::Microsoft.Extensions.Logging.EventId({lm.EventId}, {eventName}),
                 {nestedIndentation}{enabledCheckIndentation}");
-                GenHolder(lm);
-                _builder.Append($@",
+                    GenHolder(lm);
```

Comment:
@dotnet/area-extensions-logging do you know if the CI failures might be related to changes to the gen files in this PR? Some of the failures say that lines do not match, but I don't know enough to determine if we need to avoid changes in such files.

Reply:
@carlossanlop Should I just undo the changes in this file so we can merge?

Reply:
Seems that resetting that file did the trick.